### DISCUSSION TITLE
Can I safely use the Handlebars helpers?

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#12074] [Request Help] [answered]
The page on [templates](https://docs.renovatebot.com/templates/) documents the fields available in the templates, but does not mention the registered helpers `encodeURIComponent` and `replace`. Are these functions part of the contract? Can I safely and reliably use them in my own Renovate configuration?

**Comment by rarkins** [comment, marked as answer]
Good question. We should make them part of the "contract" just in case but yes, you can assume we won't remove them

**Reply by astellingwerf** [reply]
#12643 added a documentation section where this would fit.

**Reply by HonkingGoose** [reply]
This discussion will be resolved by PR #12672. 😉
